.eslintrc.js

```diff
@@ -1,19 +1,8 @@
 module.exports = {
-  env: {
-    es6: true,
-    'shared-node-browser': true,
-  },
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: './tsconfig.json',
-    sourceType: 'module',
-    ecmaVersion: 2018,
-    ecmaFeatures: {
-      jsx: true,
-    },
-  },
-  settings: {
-    'import/ignore': ['node_modules'],
+  extends: '../shared-configs/eslint-lib',
+  rules: {
+    'no-restricted-imports': ['error', { patterns: ['./*', '../*'] }],
```

Comment:
`rxbeach` used a fairly recent copied-over config from `ardoq-front`. Here, I just kept the rules that existed in addition to the standard ones. This one I guess is to force "clean" imports using `rxbeach/`.